tst/EnergyPlus/unit/AirTerminalSingleDuct.unit.cc

```diff
@@ -1059,18 +1059,18 @@ TEST_F(EnergyPlusFixture, SingleDuctVAVReheatAirTerminal_MinFlowTurnDownTest)
     DataLoopNode::Node(InletNodeNum).MassFlowRateMaxAvail = SysMaxMassFlowRes;
     DataGlobals::BeginEnvrnFlag = true;
     FirstHVACIteration = true;
-    SingleDuct::sd_airterminal(SysNum).InitSys(SysNum, FirstHVACIteration);
+    SingleDuct::sd_airterminal(SysNum).InitSys(FirstHVACIteration);
     DataGlobals::BeginEnvrnFlag = false;
     FirstHVACIteration = false;
-    SingleDuct::sd_airterminal(SysNum).InitSys(SysNum, FirstHVACIteration);
-    SingleDuct::sd_airterminal(SysNum).SimVAV(SysNum, FirstHVACIteration, ZoneNum, ZoneNodeNum);
+    SingleDuct::sd_airterminal(SysNum).InitSys(FirstHVACIteration);
+    SingleDuct::sd_airterminal(SysNum).SimVAV(FirstHVACIteration, ZoneNum, ZoneNodeNum);
```

Comment:
Removed index argement from unit tests. You will see the same changes next in several places.